decidim-budgets_booth/app/controllers/concerns/decidim/budgets_booth/line_items_controller_extensions.rb

```diff
@@ -16,7 +16,8 @@ module LineItemsControllerExtensions
         def set_help_modal
           rule_enabled = current_component[:settings]["global"]["vote_rule_selected_projects_enabled"]
           minimum_vote = current_component[:settings]["global"]["vote_selected_projects_minimum"]
-          line_items = Decidim::Budgets::Order.find_by(user: current_user, budget: budget)&.line_items
+          order ||= Decidim::Budgets::Order.find_by(user: current_user, budget: budget)
```

Comment:
```suggestion
def order
    @order ||= Decidim::Budgets::Order.find_by(user: current_user, budget: budget)
end
```

Reply:
hi @Quentinchampenois , thanks for you request ! 
I have updated the code with your suggestion